sources/updater/waves/accelerated-waves.toml

```
@@ -0,0 +1,24 @@
+# NOTE: Update waves have an implicit start time. The initial wave starts
+# "now" or at the time of deployment and includes all seeds up to but not
+# including the `end_seed`. The first wave ends and the second wave begins
+# starting at the `end_time` of the first wave. The second wave includes
+# all seeds from the first two waves, and so on and so forth.
+
+# The following represents an "accelerated" set of update waves for a much
+# quicker deployment. The deployment lasts for 1 day, and quickly increases the
+# nodes updated at once.
+[[wave]]
+end_time = 'in 2 hours'
+end_seed = 64
+
+[[wave]]
+end_time = 'in 4 hours'
+end_seed = 512
```

```suggestion
end_seed = 256
```

This was a much bigger jump, a different curve shape, and I wasn't sure why.